libr/core/disasm.c

```diff
@@ -2572,8 +2572,15 @@ static void ds_print_asmop_payload(RDisasmState *ds) {
 			break;
 		}
 	}
-	if (ds->asmop.payload != 0)
+	if (ds->asmop.payload != 0) {
 		r_cons_printf ("\n; .. payload of %d bytes", ds->asmop.payload);
+		if (r_config_get_i (ds->core->config, "asm.showpayloads")) {
```

Comment:
Cache this var in ds_init and just use the value here to avoid resolving it on every iteration